Makefile

```diff
@@ -246,13 +250,10 @@ genproto_sub:
 	@rm -rf $(OPENTELEMETRY_PROTO_SRC_DIR)/*
 	@rm -rf $(OPENTELEMETRY_PROTO_SRC_DIR)/.* > /dev/null 2>&1 || true
 
-remove-pdatagen:
-	rm -f .tools/pdatagen
```

Comment:
We do want to force recompilation of pdatagen whenever it's executed. It's a bit of a different tool than the others, since it's located within this repository, and has definitions for code generation.